# Dataset Exploration

## Setup

In [1]:
library(arrow)
library(tidyverse)


Attaching package: ‘arrow’


The following object is masked from ‘package:utils’:

    timestamp


── Attaching packages ─────────────────────────────────────── tidyverse 1.3.0 ──

✔ ggplot2 3.3.2     ✔ purrr   0.3.4
✔ tibble  3.0.3     ✔ dplyr   1.0.0
✔ tidyr   1.1.0     ✔ stringr 1.4.0
✔ readr   1.3.1     ✔ forcats 0.5.0

── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()



In [2]:
df <- read_feather("data/output.feather")
head(df)

name                                   year boardid isin         cusip    
1 SCHLUMBERGER LIMITED                   2015   27242 AN8068571086 806857108
2 SCHLUMBERGER LIMITED                   2016   27242 AN8068571086 806857108
3 SCHLUMBERGER LIMITED                   2017   27242 AN8068571086 806857108
4 SCHLUMBERGER LIMITED                   2018   27242 AN8068571086 806857108
5 ONEMARKET LTD (De-listed 12/2019)      2018 2822352 AU0000008641 000000864
6 AMCOR PLC (Amcor Ltd prior to 06/2019) 2015    1736 AU000000AMC4 000000AMC
  gvkey  cik        number_board_members number_board_members_fm
1 009465 0000087347 10                   3                      
2 009465 0000087347 11                   3                      
3 009465 0000087347 12                   3                      
4 009465 0000087347 12                   4                      
5 NA     NA          4                   1                      
6 NA     NA          8                   2                      
  number_nominating ⋯ gind   gsector avg_compensation_na_rm
1  5                ⋯ 101010 10      300.3000              
2  6                ⋯ 101010 10      299.0909              
3  5                ⋯ 101010 10      292.7500              
4  5                ⋯ 101010 10      288.0000              
5 NA                ⋯ NA     NA           NaN              
6  4                ⋯ NA     NA        0.0000              
  avg_compensation_fm_na_rm equity_compensation_na_rm
1 32.80000                  2618.222                 
2 33.45455                  2700.091                 
3 32.83333                  1741.100                 
4 40.00000                  1724.455                 
5      NaN                       NaN                 
6  0.00000                       NaN                 
  equity_compensation_fm_na_rm avg_age_na_rm members_check age_diversity_check
1 292.8889                     60.76388      10                 NA            
2 351.5455                     60.99993      11                 NA            
3 356.4000                     60.87314      12                 NA            
4 205.8182                     61.80390      12            8.71961            
5      NaN                          NaN       4                 NA            
6      NaN                     57.60799       8                 NA            
  compustat_date
1 2015-12-31    
2 2016-12-31    
3 2017-12-31    
4 2018-12-31    
5 <NA>          
6 <NA>

## Duplicate observations

In [3]:
nrow(df)

[1] 22073

In [4]:
nrow(unique(select(df, year, boardid)))

[1] 20197

Some organizations in the dataset have issued multiple securities, 1,876 duplicates with potentially diverging data should be removed.

## BoardEx members present

In [5]:
table(df$number_board_members - df$members_check)


   -2    -1     0 
    1    15 22057 

Great, only in very few cases does the number of individual entries diverge from the number of board members as provided by BoardEx.

## Number of observations

In [6]:
nrow(subset(df, !is.na(avg_compensation_fm)))
nrow(subset(df, !is.na(avg_compensation_fm_na_rm)))

[1] 2785

[1] 2785

In [7]:
nrow(subset(df, !is.na(gvkey)))
nrow(subset(df, !is.na(gvkey) & !is.na(avg_compensation_fm)))

[1] 18289

[1] 2252

Only for 2,785 company-year observations is there data available on compensation of the female members of the board (and in all cases, data is available on all female members). For 3,784 observations, merging with Compustat failed. For 2,252 observations, data is available on both female compensation, and the variables extracted from Compustat.